tests/api-testing/tests/test_foldersnew.py

```
@@ -128,12 +199,62 @@
+    
+    # Use create_session instead of session
+    resp_create_new_alerts_folders_blank = create_session.post(
+        f"{base_url}api/v2/{org_id}/folders/alerts", json=payload
+    )
+    print(create_session.headers)
+    assert resp_create_new_alerts_folders_blank.status_code == 400, (
+        f"Expected 400, but got {resp_create_new_alerts_folders_blank.status_code} "
+        f"{resp_create_new_alerts_folders_blank.content}"
+    )
+
+def test_retrieve_alerts_folders_400(create_session, base_url):
```

Ruff (0.8.2)

Redefinition of unused `test_retrieve_alerts_folders_400` from line 213

(F811)